contributors/design-proposals/seccomp.md

```diff
@@ -214,6 +248,192 @@ profiles using the key
+| `stime`             | Time/date is not namespaced. Also gated by `CAP_SYS_TIME`.                                                    |
+| `swapon`            | Deny start/stop swapping to file/device. Also gated by `CAP_SYS_ADMIN`.                                       |
+| `swapoff`           | Deny start/stop swapping to file/device. Also gated by `CAP_SYS_ADMIN`.                                       |
+| `sysfs`             | Obsolete syscall.                                                                                             |
+| `_sysctl`           | Obsolete, replaced by /proc/sys.                                                                              |
+| `umount`            | Should be a privileged operation. Also gated by `CAP_SYS_ADMIN`.                                              |
+| `umount2`           | Should be a privileged operation. Also gated by `CAP_SYS_ADMIN`.                                              |
+| `unshare`           | Deny cloning new namespaces for processes. Also gated by `CAP_SYS_ADMIN`, with the exception of `unshare --user`. |
+| `uselib`            | Older syscall related to shared libraries, unused for a long time.                                            |
+| `userfaultfd`       | Userspace page fault handling, largely needed for process migration.                                          |
+| `ustat`             | Obsolete syscall.                                                                                             |
+| `vm86`              | In kernel x86 real mode virtual machine. Also gated by `CAP_SYS_ADMIN`.                                       |
+| `vm86old`           | In kernel x86 real mode virtual machine. Also gated by `CAP_SYS_ADMIN`.                                       |
+
+#### Default Behavior
+
+For `privileged` containers, no default seccomp profile will be used unless
+explicitly requested by the user via annotations.
+
+If `capAdd` is used on a Container, the default profile will be adjusted to
+interact accordingly with the capability added. These are documented below in
+a table by the cap being added:
+
+| Capability           | Syscalls Allowed                                             |
+|----------------------|--------------------------------------------------------------|
+| `CAP_CHOWN`          | chown, chown32, fchown, fchown32, fchownat, lchown, lchown32 |
+| `CAP_DAC_READ_SEARCH`| open_by_handle_at                                            |
+| `CAP_IPC_LOCK`       | mlock, mlock2, mlockall                                      |
+| `CAP_SYS_ADMIN`      | name_to_handle_at, bpf, clone, fanotify_init, lookup_dcookie, mount, perf_event_open, setdomainname, sethostname, setns, umount, umount2, unshare |
+| `CAP_SYS_BOOT`       | reboot                                                       |
```

Comment:
it's been in docker's default and there have been no complaints, do _you_ reboot in a container, where is your cause for concern, just curious